compiler/src/parsing/well_formedness.re

```diff
@@ -6,7 +6,6 @@ type wferr =
   | MalformedString(Location.t)
   | IllegalCharacterLiteral(string, Location.t)
   | MultipleModuleName(Location.t)
-  | TypeNameShouldBeUppercase(string, Location.t)
```

Comment:
This was never actually used because the parser disallows you from using the wrong kind of identifier (but gives a nice error message telling you that you should have used an uppercase identifier).